native/storybook/rtl-addon/register.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import React from 'react';
+import addons from '@storybook/addons';
+import { RTL_EVENT } from './../constants';
+
+class Panel extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      rtlEnabled: false,
+    };
+  }
+  toggleRtl = () => {
+    this.setState({
+      rtlEnabled: !this.state.rtlEnabled,
+    });
+    const channel = addons.getChannel();
+    channel.emit(RTL_EVENT, this.state.rtlEnabled);
+  };
+  render() {
+    return (
+      <div>
+        <button onClick={this.toggleRtl}>
```

Comment:
Can we use a `BpkButton` here?

Reply:
The thought process here was to keep it vanilla because it's just rendering inside the storybook, plus in the near future we could pull this out into its own repo and use it as a plugin.

Having said that, happy to change if consensus goes the other way though.

Reply:
Okay fair enough then